src/client/pages/_slug.vue

```diff
@@ -24,3 +30,51 @@ export default {
   computed: {},
 }
 </script>
+
+<style>
+.page {
+  margin-top: 80px;
+  display: grid;
+  grid-template-columns: 1fr 3fr;
+  grid-template-areas:
+    "header header"
+    "sidebar content";
+}
+
+.page-header {
+  grid-area: header;
+  height: 250px;
```

Comment:
Don't use a fixed height. This causes issues if people change their font size settings. Make sure the size of the page header is defined by its content.